infrastructure/tooling/src/dev/helm.js

```diff
@@ -22,9 +29,26 @@ const actions = [
       if (!config.meta || !config.meta.deploymentPrefix) {
         throw new Error('Must have configured dev-config.yml to deploy.');
       }
+
+      if (config.auth && config.auth.static_clients) {
+        if (config.auth.static_clients.some(({scopes}) => Boolean(scopes))) {
+          throw new Error('auth.static_clients in `dev-config.yml` should not contain scopes');
+        }
+      }
       return {'dev-config': config};
     },
   },
+  {
+    title: 'Get static clients',
+    requires: ['dev-config'],
+    provides: ['static-clients'],
+    run: async (requirements, utils) => {
+      const ymlFile = await readRepoYAML('infrastructure/tooling/static-clients.yml');
+      const config = requirements['dev-config'];
+      const staticClients = await jsone(ymlFile, {azureAccountId: config.azureAccountId || 'unknown'});
```

Comment:
Do we actually want to fallback to `unknown` here or just bail?

Reply:
I don't know if this gets run in a case where this value isn't yet determined.  If not, I can remove the fallback.